(bug): Continuous Integration Tests fails

**Describe the bug**
Continuous Integration Tests via Github Actions fails for new pull requests.

**Steps to reproduce the behavior:**
1. Create a new pull request
2. See Error in GitHub Actions

**Expected behavior**
If GitHub Actions are activated they should work without errors per default. Otherwise disable these.

**Screenshots**
If applicable, add screenshots to help explain your problem.
![image](https://user-images.githubusercontent.com/17799657/79841079-2f0f3100-83b7-11ea-890b-5f422cf60747.png)

**Affected version (please complete the following information):**
N/A

**Log Output (ensure to remove any confidential information like tokens, project names, etc.**
```sh
Run oc cluster up --base-dir=${HOME}/openshift.local.clusterup --routing-suffix 172.17.0.1.nip.io --public-hostname 172.17.0.1 --enable=centos-imagestreams --enable=persistent-volumes --enable=registry --enable=router
  oc cluster up --base-dir=${HOME}/openshift.local.clusterup --routing-suffix 172.17.0.1.nip.io --public-hostname 172.17.0.1 --enable=centos-imagestreams --enable=persistent-volumes --enable=registry --enable=router
  shell: /bin/bash -e {0}
  env:
    GOROOT: /opt/hostedtoolcache/go/1.13.0/x64
Getting a Docker client ...
Checking if image openshift/origin-control-plane:v3.11 is available ...
Pulling image openshift/origin-control-plane:v3.11
E0421 07:45:17.925885    3550 helper.go:179] Reading docker config from /home/runner/.docker/config.json failed: open /home/runner/.docker/config.json: no such file or directory, will attempt to pull image docker.io/openshift/origin-control-plane:v3.11 anonymously
Pulled 2/5 layers, 40% complete
Pulled 3/5 layers, 71% complete
Pulled 4/5 layers, 83% complete
Pulled 5/5 layers, 100% complete
Extracting
Image pull complete
Pulling image openshift/origin-cli:v3.11
E0421 07:45:29.263750    3550 helper.go:179] Reading docker config from /home/runner/.docker/config.json failed: open /home/runner/.docker/config.json: no such file or directory, will attempt to pull image docker.io/openshift/origin-cli:v3.11 anonymously
Image pull complete
E0421 07:45:29.813536    3550 helper.go:179] Reading docker config from /home/runner/.docker/config.json failed: open /home/runner/.docker/config.json: no such file or directory, will attempt to pull image docker.io/openshift/origin-node:v3.11 anonymously
Pulling image openshift/origin-node:v3.11
Pulled 6/6 layers, 100% complete
Extracting
I0421 07:45:49.040761    3550 run_kubelet.go:49] Running "start-kubelet"
I0421 07:45:49.201138    3550 run_self_hosted.go:181] Waiting for the kube-apiserver to be ready ...
I0421 07:46:12.212277    3550 interface.go:26] Installing "kube-proxy" ...
I0421 07:46:12.212305    3550 interface.go:26] Installing "kube-dns" ...
I0421 07:46:12.212311    3550 interface.go:26] Installing "openshift-service-cert-signer-operator" ...
I0421 07:46:12.212317    3550 interface.go:26] Installing "openshift-apiserver" ...
I0421 07:46:12.212339    3550 apply_template.go:81] Installing "openshift-apiserver"
I0421 07:46:12.212483    3550 apply_template.go:81] Installing "kube-proxy"
I0421 07:46:12.212656    3550 apply_template.go:81] Installing "kube-dns"
I0421 07:46:12.212790    3550 apply_template.go:81] Installing "openshift-service-cert-signer-operator"
E0421 07:46:44.117968    3550 interface.go:34] Failed to install "openshift-service-cert-signer-operator": failed to install "openshift-service-cert-signer-operator": cannot create container using image openshift/origin-cli:v3.11; caused by: cannot create container using image openshift/origin-cli:v3.11
E0421 07:46:44.119410    3550 interface.go:34] Failed to install "kube-dns": failed to install "kube-dns": cannot create container using image openshift/origin-cli:v3.11; caused by: cannot create container using image openshift/origin-cli:v3.11
E0421 07:46:44.120208    3550 interface.go:34] Failed to install "kube-proxy": failed to install "kube-proxy": cannot create container using image openshift/origin-cli:v3.11; caused by: cannot create container using image openshift/origin-cli:v3.11
E0421 07:46:44.123463    3550 interface.go:34] Failed to install "openshift-apiserver": failed to install "openshift-apiserver": cannot create container using image openshift/origin-cli:v3.11; caused by: cannot create container using image openshift/origin-cli:v3.11
I0421 07:46:44.123488    3550 interface.go:41] Finished installing "kube-proxy" "kube-dns" "openshift-service-cert-signer-operator" "openshift-apiserver"
Error: [failed to install "openshift-service-cert-signer-operator": cannot create container using image openshift/origin-cli:v3.11; caused by: cannot create container using image openshift/origin-cli:v3.11, failed to install "kube-dns": cannot create container using image openshift/origin-cli:v3.11; caused by: cannot create container using image openshift/origin-cli:v3.11, failed to install "kube-proxy": cannot create container using image openshift/origin-cli:v3.11; caused by: cannot create container using image openshift/origin-cli:v3.11, failed to install "openshift-apiserver": cannot create container using image openshift/origin-cli:v3.11; caused by: cannot create container using image openshift/origin-cli:v3.11]
##[error]Process completed with exit code 1.
```

**Additional context**
https://github.com/opendevstack/ods-quickstarters/blob/4d67d19bd79c519c1d926ac0d91a297ea7a99053/.github/workflows/continuous-integration-workflow.yml#L74-L79

--- a/.github/workflows/continuous-integration-workflow.yml
+++ b/.github/workflows/continuous-integration-workflow.yml
@@ -1,93 +1,93 @@
-name: Continous Integration Tests
-on: [ push, pull_request ]
-jobs:
-  test:
-    name: Docker plain provisioning
-    runs-on: ubuntu-16.04
-    steps:
-      - 
-        name: GitHub context
-        env:
-          GITHUB_CONTEXT: ${{ toJson(github) }}
-        run: jq . <<< "${GITHUB_CONTEXT}"
-      - 
-        name: Checkout QuickStarters repository
-        uses: actions/checkout@v2.0.0
-        with:
-          fetch-depth: 0
-          path: ods-quickstarters
-      - 
-        name: Checkout ODS Core
-        uses: actions/checkout@v2.0.0
-        with:
-          fetch-depth: 0
-          repository: opendevstack/ods-core
-          ref: master
-          path: ods-core
-      - 
-        name: Checkout ODS Jenkins Shared Library
-        uses: actions/checkout@v2.0.0
-        with:
-          fetch-depth: 0
-          repository: BIX-Digital/ods-jenkins-shared-library
-          ref: invalid/curl-http-only
-          path: ods-jenkins-shared-library
-      -
-        name: Setup Go 1.13
-        uses: actions/setup-go@v1.0.0
-        with:      
-          version: 1.13
-      -
-        name: Download OpenShift Client
-        run: |
-          wget https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz
-          tar -xzvf openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz
-          sudo mv openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit/oc /usr/local/bin/oc
-      -
-        name: Download Tailor
-        run: |
-          curl -LO "https://github.com/opendevstack/tailor/releases/download/v0.11.0/tailor-linux-amd64"
-          chmod +x tailor-linux-amd64
-          sudo mv tailor-linux-amd64 /usr/local/bin/tailor
-      -
-         name: Tailor version
-         run: tailor version
-      -
-         name: OpenShift client version
-         run: oc version
-      -
-         name: jq version
-         run: jq --version
-      -
-         name: golang version
-         run: go version
-      -
-         name: Docker version
-         run: docker --version
-      -
-         name: Network before changes
-         run: ifconfig
-      -
-         name: Configure docker network and insecure registries
-         run: ls -la && ls -la ./tests && ./tests/scripts/apply-docker-settings.sh
-         working-directory: ${{  format('{0}/ods-core', github.workspace) }}
-      -
-         name: Start OC cluster
-         run: oc cluster up --base-dir=${HOME}/openshift.local.clusterup --routing-suffix 172.17.0.1.nip.io --public-hostname 172.17.0.1 --enable=centos-imagestreams --enable=persistent-volumes --enable=registry --enable=router
-      -
-         name: Login into the cluster
-         run: oc login -u system:admin
-      -
-        name: Create test infrastructure
-        run: |
-          mkdir -p ods-config
-          ./tests/scripts/recreate-test-infrastructure.sh
-        working-directory: ${{  format('{0}/ods-core', github.workspace) }}
-      -
-        name: Test setup
-        run: make setup-tests
-        working-directory: ${{  format('{0}/ods-quickstarters/tests', github.workspace) }}
-      -
-        name: Run tests
-        run: make test
-        working-directory: ${{  format('{0}/ods-quickstarters/tests', github.workspace) }}
+# name: Continous Integration Tests
+# on: [ push, pull_request ]
+# jobs:
+#   test:
+#     name: Docker plain provisioning
+#     runs-on: ubuntu-16.04
+#     steps:
+#       - 
+#         name: GitHub context
+#         env:
+#           GITHUB_CONTEXT: ${{ toJson(github) }}
+#         run: jq . <<< "${GITHUB_CONTEXT}"
+#       - 
+#         name: Checkout QuickStarters repository
+#         uses: actions/checkout@v2.0.0
+#         with:
+#           fetch-depth: 0
+#           path: ods-quickstarters
+#       - 
+#         name: Checkout ODS Core
+#         uses: actions/checkout@v2.0.0
+#         with:
+#           fetch-depth: 0
+#           repository: opendevstack/ods-core
+#           ref: master
+#           path: ods-core
+#       - 
+#         name: Checkout ODS Jenkins Shared Library
+#         uses: actions/checkout@v2.0.0
+#         with:
+#           fetch-depth: 0
+#           repository: BIX-Digital/ods-jenkins-shared-library
+#           ref: invalid/curl-http-only
+#           path: ods-jenkins-shared-library
+#       -
+#         name: Setup Go 1.13
+#         uses: actions/setup-go@v1.0.0
+#         with:      
+#           version: 1.13
+#       -
+#         name: Download OpenShift Client
+#         run: |
+#           wget https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz
+#           tar -xzvf openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz
+#           sudo mv openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit/oc /usr/local/bin/oc
+#       -
+#         name: Download Tailor
+#         run: |
+#           curl -LO "https://github.com/opendevstack/tailor/releases/download/v0.11.0/tailor-linux-amd64"
+#           chmod +x tailor-linux-amd64
+#           sudo mv tailor-linux-amd64 /usr/local/bin/tailor
+#       -
+#          name: Tailor version
+#          run: tailor version
+#       -
+#          name: OpenShift client version
+#          run: oc version
+#       -
+#          name: jq version
+#          run: jq --version
+#       -
+#          name: golang version
+#          run: go version
+#       -
+#          name: Docker version
+#          run: docker --version
+#       -
+#          name: Network before changes
+#          run: ifconfig
+#       -
+#          name: Configure docker network and insecure registries
+#          run: ls -la && ls -la ./tests && ./tests/scripts/apply-docker-settings.sh
+#          working-directory: ${{  format('{0}/ods-core', github.workspace) }}
+#       -
+#          name: Start OC cluster
+#          run: oc cluster up --base-dir=${HOME}/openshift.local.clusterup --routing-suffix 172.17.0.1.nip.io --public-hostname 172.17.0.1 --enable=centos-imagestreams --enable=persistent-volumes --enable=registry --enable=router
+#       -
+#          name: Login into the cluster
+#          run: oc login -u system:admin
+#       -
+#         name: Create test infrastructure
+#         run: |
+#           mkdir -p ods-config
+#           ./tests/scripts/recreate-test-infrastructure.sh
+#         working-directory: ${{  format('{0}/ods-core', github.workspace) }}
+#       -
+#         name: Test setup
+#         run: make setup-tests
+#         working-directory: ${{  format('{0}/ods-quickstarters/tests', github.workspace) }}
+#       -
+#         name: Run tests
+#         run: make test
+#         working-directory: ${{  format('{0}/ods-quickstarters/tests', github.workspace) }}

